src/txs/wasm/InstantiateContractForm.tsx

```diff
@@ -80,14 +78,14 @@ const InstantiateContractForm = () => {
 
   /* fee */
   const estimationTxValues = useMemo(() => values, [values])
-  const taxes = calcTaxes(coins, taxParams)
 
   const tx = {
     initialGasDenom,
     estimationTxValues,
-    taxes,
+    coins,
     createTx,
     onSuccess: { label: t("Contract"), path: "/contract" },
+    preventTax: false,
```

Comment:
Actually `preventTax` was only for the on-chain swap tx. Very unusual case.
I want to not declare `preventTax` every tx.

Reply:
Inverted the logic as discussed into a whitelist. The new prop is now called taxRequired, and has a default value of false.